jsonnet/kube-prometheus/kube-prometheus-thanos.libsonnet

```diff
@@ -5,7 +5,7 @@ local servicePort = k.core.v1.service.mixin.spec.portsType;
 {
   _config+:: {
     versions+:: {
-      thanos: 'v0.3.2',
+      thanos: 'v0.5.0',
```

Comment:
have you tried this? I think this needs an update of the prometheus-operator as well to work

Reply:
No its not fully tested yet. I am in the process of doing this now. Maybe I should add "WIP" into headline or close this until I have fully verified this. Sorry this is my first PR here. What do do you prefer?

Reply:
Feel free to leave open with WIP just let us know when you've verified it and want us to review :)

Reply:
yes will do, thanks!